templates/default/activities.html.twig

```diff
@@ -47,10 +47,10 @@
                 {% set aid = activity.getId() %}
                 <a href="{{ url_for("events_activity_edit", {"id": aid}) }}">{{ activity.getName() }}</a>
             </td>
-            <td data-title="{{ _T("Creation date", "events") }}">{{ activity.getCreationDate() }}</td>
-            <td data-title="{{ _T("Events", "events") }}">{{ activity.countEvents() }}</td>
-            <td class="center id_row {% if activity.isActive() %}use{% else %}delete{% endif %}" data-title="{{ _T("Is active", "events") }}">
-                <i class="toggle {% if activity.isActive() %}on{% else %}off{% endif %} icon"></i>
+            <td>{{ activity.getCreationDate() }}</td>
```

Comment:
I suppose it had something to do with jquery-ui.

Now with FUI, this data attribute creates conflicts with tooltips display.

So, on the contrary, I think it can also be safely removed from core as FUI doesn't use it when displaying stacked tables on mobile.

Reply:
> I suppose it had something to do with jquery-ui.

As far as I remember, this was not specifically related to jquery-ui; just something I made to make lists "usable" on a mobile view.

> Now with FUI, this data attribute creates conflicts with tooltips display.
> 
> So, on the contrary, I think it can also be safely removed from core as FUI doesn't use it when displaying stacked tables on mobile.

I need to think about that... I did not pay attention before, but new display is quite difficult to understand on mobile (it's quite impossible to know what the data is)

New display:
![image](https://github.com/galette/plugin-events/assets/224733/0adfd409-5e98-420c-ab4e-638caf4c72b3)

Old one:
![image](https://github.com/galette/plugin-events/assets/224733/0ce0a9e4-baf3-40fa-a610-80ae7f45fe40)

In the above example, only few columns are self explanatory; most of them benefits from a "label".

I have no idea how other apps proceed (nor what's the "SUI/FUI way to achieve that, if even it's possible).

Reply:
> As far as I remember, this was not specifically related to jquery-ui; just something I made to make lists "usable" on a mobile view.

All right! Then it could be used again the same way. Just need to change the data attribute name :smile: 

> I need to think about that... I did not pay attention before, but new display is quite difficult to understand on mobile (it's quite impossible to know what the data is)
> 
> [...]
> 
> I have no idea how other apps proceed (nor what's the "SUI/FUI way to achieve that, if even it's possible).

It's not a big deal actually.
I'll have a look. I think it just needs a few CSS to be done :wink:

Reply:
Done! Now requires https://github.com/galette/galette/pull/378

Reply:
G-R-E-A-T, thanks :)